shell/client/shell.html

```diff
@@ -564,14 +564,19 @@ <h4 class="share-with-others">Share with others</h4>
 
 </template>
 
-<template name="newGrain">
+<template name="apps">
   {{>title "Apps"}}
   {{> sandstormAppList }}
 </template>
 
-<template name="selectGrain">
+<template name="grains">
   {{>title "Grains"}}
-  {{> sandstormGrainList}}
+  {{> sandstormGrainListPage}}
+</template>
+
+<template name="appDetails">
+  {{>title appTitle}}
```

Comment:
Move into sandstormAppDetails where appTitle is actually defined.

Add package dependency on sandstorm-ui-topbar.